src/Modal/index.jsx

```diff
@@ -53,10 +53,6 @@ class Modal extends React.Component {
     }
   }
 
-  componentWillUnmount() {
-    ReactDOM.unmountComponentAtNode(this.parentElement);
-  }
```

Comment:
I don't understand why this was here before. This attempts to unmount an element that this component does not own which throws an error for me.

The modal should unmount itself from the parent element automatically.